lib/stripe/resources/financial_connections/account.rb

```diff
@@ -36,33 +36,30 @@ def refresh_account(params = {}, opts = {})
       end
 
       def self.disconnect(account, params = {}, opts = {})
-        resp, opts = execute_resource_request(
-          :post,
-          format("/v1/financial_connections/accounts/%<account>s/disconnect", { account: CGI.escape(account) }),
-          params,
-          opts
+        request_stripe_object(
+          method: :post,
+          path: format("/v1/financial_connections/accounts/%<account>s/disconnect", { account: CGI.escape(account) }),
+          params: params,
+          opts: opts
         )
-        Util.convert_to_stripe_object(resp.data, opts)
       end
 
       def self.list_owners(account, params = {}, opts = {})
-        resp, opts = execute_resource_request(
-          :get,
-          format("/v1/financial_connections/accounts/%<account>s/owners", { account: CGI.escape(account) }),
-          params,
-          opts
+        request_stripe_object(
+          method: :get,
+          path: format("/v1/financial_connections/accounts/%<account>s/owners", { account: CGI.escape(account) }),
+          params: params,
+          opts: opts
         )
-        Util.convert_to_stripe_object(resp.data, opts)
       end
 
       def self.refresh_account(account, params = {}, opts = {})
-        resp, opts = execute_resource_request(
-          :post,
-          format("/v1/financial_connections/accounts/%<account>s/refresh", { account: CGI.escape(account) }),
-          params,
-          opts
+        request_stripe_object(
+          method: :post,
+          path: format("/v1/financial_connections/accounts/%<account>s/refresh", { account: CGI.escape(account) }),
+          params: params,
+          opts: opts
         )
```

Comment:
This is the static method. It will call into the static version of request_stripe_object https://github.com/stripe/stripe-ruby/pull/1071/files/dbae531c96e10aff2bfc999bf9b34a71846e535c#diff-434756fa2db23d5e029e84c7fb0cbac74f9103f73777d9ab0f530dfdd9ded3f1R27